packages/server/tests/views/profile.test.ts

```diff
@@ -60,6 +60,40 @@ describe('pds profile views', () => {
     expect(forSnapshot(danForBob.data)).toMatchSnapshot()
   })
 
+  it('updates profile', async () => {
+    const toHighlight = [sc.badges[sc.dids.bob][sc.dids.alice][1].raw]
```

Comment:
If you like, you can reference their dids like this:
```suggestion
    const toHighlight = [sc.badges[bob][alice][1].raw]
```